src/reqstool/common/jinja2.py

```diff
@@ -37,8 +37,8 @@ def load_template(loader: BaseLoader) -> Template:
             return template_env.get_template(template_name)
 
         try:
-            template_module: Package = reqstool.commands.report.templates
-            template_path: Traversable = files(template_module).joinpath("")
+            template_module: Package = reqstool.commands.report
+            template_path: PosixPath = files(template_module).joinpath("templates")
```

Comment:
Can PosixPath cause issues for potential Windows users? 
I'm ok with us not supporting Windows for now but I think we should mention it in the docs then.

Reply:
I'm not sure if it will cause any issues on a Windows machine. We could try to install it on a Windows VM, but I think that it could wait a bit.

Reply:
As long as there is a disclaimer.